src/website/src/app/documentation/demos/color/color.demo.ts

```diff
@@ -18,131 +21,6 @@ import { ClarityDocComponent } from '../clarity-doc';
 export class ColorDemo extends ClarityDocComponent {
   constructor() {
     super('color');
+    ClarityIcons.add({ copy: EssentialShapes.copy });
```

Comment:
Wait, why do we have to do this? Aren't icons added globally at the root?

Reply:
no we have `import '@clr/icons';` in main.ts, which will provide only core shapes